meetings/list.md

```diff
@@ -37,9 +37,23 @@ Covers our high-level organizational mission, values, and principles, our high-l
 
 It is led by the **`Executive Director`**.
 
-- **Date**: [every other Wednesday at **5pm CET**](https://calendar.google.com/calendar/embed?src=c_nq8hl7qsm484g1p7mfkm29jpo8%40group.calendar.google.com&ctz=America%2FLos_Angeles)
+- **Date**: [every other Wednesday](https://calendar.google.com/calendar/embed?src=c_nq8hl7qsm484g1p7mfkm29jpo8%40group.calendar.google.com&ctz=America%2FLos_Angeles)
 - **Notes and Agenda**: [google doc](https://drive.google.com/open?id=1HoNX8T8IQ1uhS2ryi1r9iS-nSbPT1b1Y7HsjosbHme8&authuser=1&usp=meetingnotes&showmeetingnotespromo=true)
 
+### Process and operations
+
+Covers our operational practices and systems for coordination and planning.
+We discuss ways that we can improve organization-wide practices to become more efficient and effective at doing our work.
+
+It is led by the **`Project Manager`**.
+
+- **Date**: [every other Tuesday](https://calendar.google.com/event?action=TEMPLATE&tmeid=NzRsbm1vc3Y4ZmlmZ2xucDE3cWI0YmxlMmNfMjAyMjEwMjVUMTMwMDAwWiBjXzRoampvdW9qZDhwc3FsOWkxYThuZDF1ZmY0QGc&tmsrc=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com)
+- [ ] 
```

Comment:
Why do we have these empty places? I feel you maybe forgot to remove these ones?